prow/plugins/plugins.go

```diff
@@ -411,6 +411,13 @@ type Heart struct {
 	// Adorees is a list of GitHub logins for members
 	// for whom we will add emojis to comments
 	Adorees []string `json:"adorees,omitempty"`
+	// CommentRegexp is the regular expression for comments
+	// made by adorees that the plugin adds emojis to.
+	// If not specified, the plugin will not add emojis to
+	// any comments.
```

Comment:
> If not specified, the plugin will not add emojis to any comments.

Added this so that backward compatibility is maintained. If we don't do this, and `CommentRegexp` is empty, it will start adding reactions to all comments.

Reply:
It might be confusing to users if they add an adoree without adding a regexp and nothing happens. I would expect a config load error in this case, but that wouldn't be backwards compatible.
/shrug